src/test/ui/consts/const-eval/issue-52442.rs

```diff
@@ -1,4 +1,4 @@
 fn main() {
     [();  { &loop { break } as *const _ as usize } ];
-    //~^ ERROR casting pointers to integers in constants is unstable
+    //~^ ERROR pointers cannot be cast to integers during const eval
```

Comment:
This is basically the same test as "issue-52023-array-size-pointer-cast.rs ", I think we should remove one of them.

Reply:
not quite, this one makes sure it's a static check.

maybe put them in the same file?

Reply:
Not sure what you mean by "static" check, but sure I'll move them together.

Reply:
I misread this test. I thought it looped endlessly if we used unleash. ignore that. just remove one of them

Reply:
Kk, done